### PR TITLE
Complete wandering market and brand without connection

### PR DESCRIPTION
**Pull request by XuMinhui**
Finish browsing the market and brand without connection

### CODE REVIEW

File: src/modules/market/Routes.tsx [hunk 1]
```diff
@@ -13,19 +11,6 @@ export const MarketRoutesConfig: { [key: string]: RouteConfiguration } = {
...
-
 export function MarketRoutes() {
-  return (
-    <PrivateRoute
-      path={MarketRoutesConfig.Market.path}
-      exact={true}
-      component={LoadableContainer}
-    />
-  );
+  return <Market />;
```
Comment (Antroll) [inline src/modules/market/Routes.tsx]: I think it's not the best dicision. If you do like that, we'll lost the feature with loadable components. So market JS will be loaded with the main scripts on the main page. Which in turn slightly slows down the loading speed of the first page.

Am I right @BigBadAlien ?

Comment (BigBadAlien) [reply src/modules/market/Routes.tsx]: You are right. All routes must be loadable.

```
import { Route } from 'react-router-dom';
...
<Route
      path={MarketRoutesConfig.Market.path}
      exact={true}
      component={LoadableContainer}
    />
```

Should works

Comment (Antroll) [reply src/modules/market/Routes.tsx]: @XuMinhui please consider this comment and update this part of the code.

**Review by Antroll — CHANGES_REQUESTED** [review]

File: src/modules/market/Routes.tsx (hunk 1 above)
Comment (BigBadAlien) [reply src/modules/market/Routes.tsx]: @XuMinhui Just to clarify, we need it for optimization purposes, every page must have its own bundle to speed up initial loading time

**Comment by XuMinhui** [comment]
Okay, thank you both for the correction, this is indeed my mistake, did not take this into account, I have added, you see if it meets

**Review by Antroll — APPROVED** [review]

**Pull request merged**